DatWeatherDoe/Resources/Localization/fr.lproj/Localizable.strings

```diff
@@ -85,6 +85,9 @@
 /* Lat/Long error when fetching weather */
 "Lat/Long Error" = "Erreur de Lat/Lon";
 
+/* Launch app at login */
+"Launch at Login" = "Lancer à la connexion";
```

Comment:
@Marc66 I just added this string (as part of 2.0 coming up). Please let me know how this looks 🙏